coniferest/aadforest.py

```diff
@@ -207,14 +214,22 @@ def fit_known(self, data, known_data=None, known_labels=None):
         # Our scores are negative, so we need to "invert" the quantile.
         q_tau = np.quantile(scores, 1.0 - self.tau)
 
+        anomaly_count = np.count_nonzero(known_labels == Label.ANOMALY)
+        nominal_count = np.count_nonzero(known_labels == Label.REGULAR)
+
+        if isinstance(self.prior_influence, Callable):
+            prior_influence = self.prior_influence(anomaly_count, nominal_count)
+        else:
```

Comment:
I would move this check to the constructor and merge it with the validation. In the constructor we can do something like
```python
from numbers import Real

...

if isinstance(prior_influence, Callable):
    self.prior_influence = prior_influence
elif isinstance(prior_influence, Real):
    self.prior_influence = lambda ac, nc: prior_influence
else:
    raise ValueError(...)
```